File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_member_relations_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusMemberRelationsVO:
    """Member Relations VO"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    uuid: str = None
    member_id: int = None
    owner: str = None
    owner_id: int = None
    type: str = None
    parent_id: int = None
    target_id: int = None
    is_primary: bool = None
    is_active: bool = None
    effective_at: str = None
    expired_at: str = None
    sort_order: int = None
    remark: str = None

File: sdkwork-backend-sdk-go/types/plus_member_relations_vo.go
package types

// Member Relations VO
type PlusMemberRelationsVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	MemberId int `json:"memberId"`
	Owner string `json:"owner"`
	OwnerId int `json:"ownerId"`
	Type string `json:"type"`
	ParentId int `json:"parentId"`
	TargetId int `json:"targetId"`
	IsPrimary bool `json:"isPrimary"`
	IsActive bool `json:"isActive"`
	EffectiveAt string `json:"effectiveAt"`
	ExpiredAt string `json:"expiredAt"`
	SortOrder int `json:"sortOrder"`
	Remark string `json:"remark"`
}
